src/frontend/komponenter/Felleskomponenter/Hendelsesoversikt/BrevModul/Brevskjema.tsx

```diff
@@ -157,6 +158,30 @@ const Brevskjema = ({ onSubmitSuccess }: IProps) => {
         åpenBehandling.status === RessursStatus.SUKSESS && åpenBehandling.data.behandlingId;
 
     const fritekstSkjemaGruppeId = 'Fritekster-brev';
+
+    const fritekstHjelpetekster: Record<Brevmal, string> = {
```

Comment:
Vet vi om vi skal ha flere hjelpetekster i framtiden? Om ikke er det kanskje like så enkelt å lage den ene i en variabel `const hjelpetekstVarselAnnenForelderMedSelvstendigRettSøkt = 'Skriv her hvilke opplysninger vi har som er av betydning for saken. For eksempel: Vi har fått opplyst at barnet bor fast sammen med den andre forelderen.'`

Reply:
Sjekket med Anna, og vi skal foreløpig bare ha denne hjelpeteksten. Så jeg endrer det til som suggesta 🤩